src/foam/support/view/MessageCard.js

```diff
@@ -54,12 +60,32 @@ foam.CLASS({
     letter-spacing: 0.2px;
     text-align: left;
     color: #093649;
+    padding: 60px 928px 324px 80px;
   }
+  ^ .person {
+    width: 40px;
+    height: 40px;
+    object-fit: contain;
+    padding: 20px 0px 0px 20px;
+  }
+ ^ #v27 {
+  display: inline-block;
+  float: left;
+ 
+ }
+ ^ #v29 {
+  display: inline-block;
+  float: left;
+ }
+ ^ #v31 {
+  padding-top: 28px;
+  padding-right: 50px;
+ }
```

Comment:
please fix indents

Reply:
we can't use the id's  since they change dramatically. Please specify class names.